### ISSUE TITLE
python 3.8 syntax warnings

### ISSUE THREAD

**Issue by modir** [takeshixx/knxmap#17]
Whenever I start knxmap I get several syntax warnings. I guess this is related to python 3.8.

```
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/targets.py:115: SyntaxWarning: "is not" with a literal. Did you mean "!="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/targets.py:134: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/targets.py:252: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/targets.py:268: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/targets.py:115: SyntaxWarning: "is not" with a literal. Did you mean "!="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/targets.py:134: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/targets.py:252: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/targets.py:268: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/bus/tunnel.py:157: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/bus/tunnel.py:159: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/bus/tunnel.py:161: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/bus/tunnel.py:287: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/bus/tunnel.py:157: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/bus/tunnel.py:159: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/bus/tunnel.py:161: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/local/lib/python3.8/site-packages/KNXmap-0.10.0-py3.8.egg/knxmap/bus/tunnel.py:287: SyntaxWarning: "is" with a literal. Did you mean "=="?

```

**Comment by modir** [comment]
My assumption is correct: https://adamj.eu/tech/2020/01/21/why-does-python-3-8-syntaxwarning-for-is-literal/

**Comment by takeshixx** [comment]
Fixed via 92f45dd42f9cdeec14e31fa5112ab14c45b37cc7, thanks!
